src/index.html

```diff
@@ -8,6 +8,45 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img
+        class="card_img"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h1 class="card_description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card_code">Product code: 195434</p>
+
+      <div class="card_review">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+        </ul>
+
+        <p class="card_review_count">Reviews: 5</p>
+      </div>
+
+      <div class="card_price">
+        <p class="card_price_text">Price:</p>
+
+        <p class="card_price_amount">$2,199</p>
+      </div>
+
+      <div class="card_button">
+        <a
+          class="card_button-buy"
+          href="/"
+          data-qa="hover"
+        >Buy</a>
```

Comment:
```suggestion
        <a
          class="card_button-buy"
          href="/"
          data-qa="hover"
        >
          Buy
        </a>
```

Reply:
In my opinion, my version is more readable and more pleasant to read ;/ I'm irritated by an inscription that doesn't stick to anything ;/